### PR TITLE
Release announcement for 1.9.1

### PR DESCRIPTION
**Pull request by jacob-delgado**
Document 1.9.1

[ ] Configuration Infrastructure
[X] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by myidpt — APPROVED** [review]
Thank you jacob!

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    - /news/announcing-1.9.1
+---
+
+This release fixes the security vulnerability described in [our March 1st, 2021 news post](/news/security/istio-security-2021-001)
+as well as bug fixes to improve robustness.
+
+This release note describes what’s different between Istio 1.9.0 and Istio 1.9.1.
+
+{{< warning >}}
+Due to the nature of the fixed security issue, at the time of publication qualification tests are still pending. These
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:18]:
```suggestion
Due to the nature of the fixed security issue, at the time of publication, qualification tests are still pending. These
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+tests will be concluded on Wednesday March 3, 2021. At such time we will either remove this warning or post known issues
+seen with this release. If you are not comfortable adopting Istio 1.9.1 before Wednesday, please read through the
+[security bulletin](/news/security/istio-security-2021-001) and follow the steps to mitigate specified in it.
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:21]:
```suggestion
[security bulletin](/news/security/istio-security-2021-001) and follow the steps to mitigate within it.
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+- **Updated** prometheus metrics to include source_cluster and destination_cluster labels by default for all scenarios. Previously, this was only enabled for multi-cluster scenarios.
+  ([Issue #30036](https://github.com/istio/istio/issues/30036))
+
+- **Updated** default access log to include `RESPONSE_CODE_DETAILS` and `CONNECTION_TERMINATION_DETAILS` for proxy version >= 1.9.
+  ([Issue #27903](https://github.com/istio/istio/issues/27903))
+
+- **Updated** Kiali addon to the latest version v1.29.
+  ([Issue #30438](https://github.com/istio/istio/issues/30438))
+
+- **Added**  `enableIstioConfigCRDs` to `base` to allow user specify whether the istio crds will be installed.  ([Issue #28346](https://github.com/istio/istio/issues/28346))
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:49]:
```suggestion
- **Added**  `enableIstioConfigCRDs` to `base` to allow users to specify whether the Istio crds will be installed.  ([Issue #28346](https://github.com/istio/istio/issues/28346))
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+- **Added** support for DestinationRule inheritance for mesh/namespace level rules. Enable feature with `PILOT_ENABLE_DESTINATION_RULE_INHERITANCE` environment variable.
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:51]:
```suggestion
- **Added** support for DestinationRule inheritance for mesh/namespace level rules. Enable feature with the `PILOT_ENABLE_DESTINATION_RULE_INHERITANCE` environment variable.
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+- **Added** support for applications that bind to their pod IP address, rather than wildcard or localhost address, through the `Sidecar` API.
+  ([Issue #28178](https://github.com/istio/istio/issues/28178))
+
+- **Added** flag to enable capture of dns traffic to the istio-iptables script.
+  ([Issue #29908](https://github.com/istio/istio/issues/29908))
+
+- **Added** canonical service tags to Envoy-generated trace spans.
+  ([Issue #28801](https://github.com/istio/istio/issues/28801))
+
+- **Fixed** an issue causing the timeout header `x-envoy-upstream-rq-timeout-ms` was not honoured.
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:63]:
```suggestion
- **Fixed** an issue causing the timeout header `x-envoy-upstream-rq-timeout-ms` to not be honoured.
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+- **Fixed** an issue where access log service causes Istio proxy to reject configuration.
+  ([Issue #30939](https://github.com/istio/istio/issues/30939))
+
+- **Fixed** an issue causing an alternative Envoy binary to be included in the docker image. The binaries are functionally equivalent.
+  ([Issue #31038](https://github.com/istio/istio/issues/31038))
+
+- **Fixed** an issue where the TLSv2 version was enforced only on HTTP ports. This option is now applied to all ports.
+
+- **Fixed** an issue where Wasm plugin configuration update will cause request to fail.
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:74]:
```suggestion
- **Fixed** an issue where Wasm plugin configuration update will cause requests to fail.
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
+---
+title: Announcing Istio 1.9.1
+linktitle: 1.9.1
+subtitle: Patch Release
+description: Istio 1.9.1 patch release.
+publishdate: 2020-12-08
```
Comment (duderino) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:6]:
```suggestion
publishdate: 2021-03-01
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md (hunk 1 above)
Comment (duderino) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md]:
```suggestion
At the time of release publication, we have only completed 18 our of our typical 48 hours of stress testing.   Given the zero day nature of the vulnerability and our experience with these stress tests, we believe it's safe to release now. These
```

Comment (duderino) [reply content/en/news/releases/1.9.x/announcing-1.9.1/index.md]: I inserted an alternative suggestion.  I don't care about the wording, but I think we should be more specific about what has and has not finished.  So I think we should somehow say that the only shortcut we are taking is only completing 18 out of 48 hours of stress testing.  All the other tests are finished.

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 8]
```diff
@@ -0,0 +1,78 @@
...
+tests will be concluded on Wednesday March 3, 2021. At such time we will either remove this warning or post known issues
+seen with this release. If you are not comfortable adopting Istio 1.9.1 before Wednesday, please read through the
+[security bulletin](/news/security/istio-security-2021-001) and follow the steps to mitigate specified in it.
+{{< /warning >}}
+
+{{< relnote >}}
+
+## Security update
+
+A 0 day was fixed in Envoy that affects only Istio 1.9.0 on Friday February 26th, 2021. This 0 day can be exploited only
```
Comment (craigbox) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:29]:
```suggestion
A zero-day security vulnerability was fixed in the version of Envoy shipped with Istio 1.9.0.  This vulnerability was fixed on Friday February 26th, 2021. 1.9.0 is the only version of Istio that includes the vulnerable version of Envoy. This vulnerability can only be exploited
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+## Changes
+
+- **Improved** sidecar injection to automatically specify the `kubectl.kubernetes.io/default-logs-container`. This ensures `kubectl logs`
+  defaults to reading the application container's logs, rather than requiring explicitly setting the container.
+  ([Issue #26764](https://github.com/istio/istio/issues/26764))
+
+- **Improved** the sidecar injector to better utilize pod labels to determine if injection is required. This is not enabled
+  by default in this release, but can be tested using `--set values.sidecarInjectorWebhook.useLegacySelectors=false`.  ([Issue #30013](https://github.com/istio/istio/issues/30013))
+
+- **Updated** prometheus metrics to include source_cluster and destination_cluster labels by default for all scenarios. Previously, this was only enabled for multi-cluster scenarios.
```
Comment (craigbox) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:41]:
```suggestion
- **Updated** Prometheus metrics to include source_cluster and destination_cluster labels by default for all scenarios. Previously, this was only enabled for multi-cluster scenarios.
```

File: content/en/news/security/istio-security-2021-001/index.md [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+vector: ""
+releases: ["1.9.0"]
+publishdate: 2021-03-1
+keywords: [CVE]
+skip_seealso: true
+---
+
+{{< security_bulletin >}}
+
+This issue only affects Istio 1.9.0. Previous versions of Istio are not affected. This is rated an 8.2 CVE by Istio.
```
Comment (duderino) [inline content/en/news/security/istio-security-2021-001/index.md:16]:
```suggestion
This issue only affects Istio 1.9.0. Previous versions of Istio are not affected. This is rated an 8.2 CVSS CVE by Istio.
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+- **Added**  `enableIstioConfigCRDs` to `base` to allow users to specify whether the Istio crds will be installed.  ([Issue #28346](https://github.com/istio/istio/issues/28346))
+
+- **Added** support for DestinationRule inheritance for mesh/namespace level rules. Enable feature with the `PILOT_ENABLE_DESTINATION_RULE_INHERITANCE` environment variable.
+  ([Issue #29525](https://github.com/istio/istio/issues/29525))
+
+- **Added** support for applications that bind to their pod IP address, rather than wildcard or localhost address, through the `Sidecar` API.
+  ([Issue #28178](https://github.com/istio/istio/issues/28178))
+
+- **Added** flag to enable capture of dns traffic to the istio-iptables script.
```
Comment (craigbox) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:58]:
```suggestion
- **Added** flag to enable capture of DNS traffic to the `istio-iptables` script.
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+- **Added** canonical service tags to Envoy-generated trace spans.
+  ([Issue #28801](https://github.com/istio/istio/issues/28801))
+
+- **Fixed** an issue causing the timeout header `x-envoy-upstream-rq-timeout-ms` to not be honored.
+  ([Issue #30885](https://github.com/istio/istio/issues/30885))
+
+- **Fixed** an issue where access log service causes Istio proxy to reject configuration.
+  ([Issue #30939](https://github.com/istio/istio/issues/30939))
+
+- **Fixed** an issue causing an alternative Envoy binary to be included in the docker image. The binaries are functionally equivalent.
```
Comment (craigbox) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:70]:
```suggestion
- **Fixed** an issue causing an alternative Envoy binary to be included in the Docker image. The binaries are functionally equivalent.
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md (hunk 1 above)
Comment (jacob-delgado) [reply content/en/news/releases/1.9.x/announcing-1.9.1/index.md]: Did a manual commit

File: content/en/news/security/istio-security-2021-001/index.md (hunk 10 above)
Comment (craigbox) [inline content/en/news/security/istio-security-2021-001/index.md]:
```suggestion
This issue only affects Istio 1.9.0; previous versions of Istio are not affected. This issue is rated an 8.2 CVE by the Istio product security working group.
```

File: content/en/news/security/istio-security-2021-001/index.md [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+You are not subject to the issue if you use both `RequestAuthentication` and `AuthorizationPolicy` for JWT validation.
+
+{{< warning >}}
+Please note that `RequestAuthentication` is used to define a list of issuers that should be accepted. It does not reject
+a request without JWT token.
+{{< /warning >}}
+
+For Istio, this vulnerability only exists if your service:
+* Accepts JWT tokens (with `RequestAuthentication`)
+* Some service paths don’t have `AuthorizationPolicy` applied.
```
Comment (craigbox) [inline content/en/news/security/istio-security-2021-001/index.md:33]:
```suggestion
* Has some service paths without `AuthorizationPolicy` applied.
```

**Review by craigbox — CHANGES_REQUESTED** [review]
ease of reading fixes

File: content/en/news/security/istio-security-2021-001/index.md (hunk 10 above)
Comment (jacob-delgado) [reply content/en/news/security/istio-security-2021-001/index.md]: Fixed manually

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 14]
```diff
@@ -0,0 +1,79 @@
...
+---
+
+This release fixes the security vulnerability described in [our March 1st, 2021 news post](/news/security/istio-security-2021-001)
+as well as bug fixes to improve robustness.
+
+This release note describes what’s different between Istio 1.9.0 and Istio 1.9.1.
+
+{{< warning >}}
+At the time of release publication, we have only completed 18 out of our typical 48 hours of stress testing. Given the
+0-day nature of the vulnerability and our experience with these stress tests, we believe it's safe to release now. These
```
Comment (craigbox) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:19]:
```suggestion
zero-day nature of the vulnerability and our experience with these stress tests, we believe it's safe to release now. These
```

missed one sorry

File: content/en/news/security/istio-security-2021-001/index.md [hunk 15]
```diff
@@ -0,0 +1,46 @@
+---
+title: ISTIO-SECURITY-2021-001
+subtitle: Security Bulletin
+description: JWT authentication can be bypassed when AuthorizationPolicy is misused.
+cves: [N/A]
+cvss: "N/A"
```
Comment (craigbox) [inline content/en/news/security/istio-security-2021-001/index.md:6]: 8.2 ?

File: content/en/news/security/istio-security-2021-001/index.md [hunk 16]
```diff
@@ -0,0 +1,46 @@
...
+This issue only affects Istio 1.9.0; previous versions of Istio are not affected. This issue has been given a CVSS score
+of 8.2 by the Istio product security working group.
+
+Envoy, and subsequently Istio, is vulnerable to a newly discovered vulnerability:
+
+- [Envoy JWT filter bypass when using the allow_missing configuration under requires_any](https://groups.google.com/g/envoy-security-announce/c/aqtBt5VUor0).
+
+You are subject to the vulnerability if you are using `RequestAuthentication` alone for JWT validation.
+
+You are not subject to the issue if you use both `RequestAuthentication` and `AuthorizationPolicy` for JWT validation.
```
Comment (craigbox) [inline content/en/news/security/istio-security-2021-001/index.md:25]:
```suggestion
You are **not** subject to the vulnerability if you use **both** `RequestAuthentication` and `AuthorizationPolicy` for JWT validation.
```

File: content/en/news/security/istio-security-2021-001/index.md [hunk 17]
```diff
@@ -0,0 +1,46 @@
+---
+title: ISTIO-SECURITY-2021-001
+subtitle: Security Bulletin
+description: JWT authentication can be bypassed when AuthorizationPolicy is misused.
+cves: [N/A]
+cvss: "N/A"
+vector: ""
+releases: ["1.9.0"]
+publishdate: 2021-03-1
```
Comment (craigbox) [inline content/en/news/security/istio-security-2021-001/index.md:9]:
```suggestion
publishdate: 2021-03-01
```

Comment (craigbox) [reply content/en/news/security/istio-security-2021-001/index.md:9]: Hopefully this will fix the fact that it's not sorting correctly in the page list.

**Review by craigbox — CHANGES_REQUESTED** [review]

File: content/en/news/security/istio-security-2021-001/index.md (hunk 15 above)
Comment (jacob-delgado) [reply content/en/news/security/istio-security-2021-001/index.md]: Added

**Review by duderino — APPROVED** [review]

**Review by craigbox — APPROVED** [review]

**Comment by ericvn** [comment]
My guess on spelling based on placement and without pulling and building locally.
       

41 | Prometheus metrics to include `source_cluster` and `destination_cluster` label 
       41 | to include `source_cluster` and `destination_cluster` labels by default for all sce 
       47 | i addon to the latest version `v1.29`. 
       50 |  to specify whether the Istio `crds` will be installed.  ([Issue # 
       52 | - **Added** support for `DestinationRule` inheritance for mesh/namespac 
       73 |  **Fixed** an issue where the `TLSv2` version was enforced only on  

    ./en/news/security/istio-security-2021-001/index.md
       21 |  filter bypass when using the `allow_missing `configuration under requires_ 
       21 | w_missing configuration under `requires_any`](https://groups.google.com/g/

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 18]
```diff
@@ -0,0 +1,79 @@
...
+## Changes
+
+- **Improved** sidecar injection to automatically specify the `kubectl.kubernetes.io/default-logs-container`. This ensures `kubectl logs`
+  defaults to reading the application container's logs, rather than requiring explicitly setting the container.
+  ([Issue #26764](https://github.com/istio/istio/issues/26764))
+
+- **Improved** the sidecar injector to better utilize pod labels to determine if injection is required. This is not enabled
+  by default in this release, but can be tested using `--set values.sidecarInjectorWebhook.useLegacySelectors=false`.  ([Issue #30013](https://github.com/istio/istio/issues/30013))
+
+- **Updated** Prometheus metrics to include source_cluster and destination_cluster labels by default for all scenarios. Previously, this was only enabled for multi-cluster scenarios.
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:41]:
```suggestion
- **Updated** Prometheus metrics to include `source_cluster` and `destination_cluster` labels by default for all scenarios. Previously, this was only enabled for multi-cluster scenarios.
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 19]
```diff
@@ -0,0 +1,79 @@
...
+  ([Issue #30036](https://github.com/istio/istio/issues/30036))
+
+- **Updated** default access log to include `RESPONSE_CODE_DETAILS` and `CONNECTION_TERMINATION_DETAILS` for proxy version >= 1.9.
+  ([Issue #27903](https://github.com/istio/istio/issues/27903))
+
+- **Updated** Kiali addon to the latest version v1.29.
+  ([Issue #30438](https://github.com/istio/istio/issues/30438))
+
+- **Added**  `enableIstioConfigCRDs` to `base` to allow users to specify whether the Istio crds will be installed.  ([Issue #28346](https://github.com/istio/istio/issues/28346))
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:50]:
```suggestion
- **Added**  `enableIstioConfigCRDs` to `base` to allow users to specify whether the Istio CRDs will be installed.  ([Issue #28346](https://github.com/istio/istio/issues/28346))
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 20]
```diff
@@ -0,0 +1,79 @@
...
+- **Fixed** an issue causing the timeout header `x-envoy-upstream-rq-timeout-ms` to not be honored.
+  ([Issue #30885](https://github.com/istio/istio/issues/30885))
+
+- **Fixed** an issue where access log service causes Istio proxy to reject configuration.
+  ([Issue #30939](https://github.com/istio/istio/issues/30939))
+
+- **Fixed** an issue causing an alternative Envoy binary to be included in the Docker image. The binaries are functionally equivalent.
+  ([Issue #31038](https://github.com/istio/istio/issues/31038))
+
+- **Fixed** an issue where the TLSv2 version was enforced only on HTTP ports. This option is now applied to all ports.
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:73]:
```suggestion
- **Fixed** an issue where the `TLSv2` version was enforced only on HTTP ports. This option is now applied to all ports.
```

File: content/en/news/releases/1.9.x/announcing-1.9.1/index.md [hunk 21]
```diff
@@ -0,0 +1,79 @@
...
+
+- **Updated** default access log to include `RESPONSE_CODE_DETAILS` and `CONNECTION_TERMINATION_DETAILS` for proxy version >= 1.9.
+  ([Issue #27903](https://github.com/istio/istio/issues/27903))
+
+- **Updated** Kiali addon to the latest version v1.29.
+  ([Issue #30438](https://github.com/istio/istio/issues/30438))
+
+- **Added**  `enableIstioConfigCRDs` to `base` to allow users to specify whether the Istio crds will be installed.  ([Issue #28346](https://github.com/istio/istio/issues/28346))
+
+- **Added** support for DestinationRule inheritance for mesh/namespace level rules. Enable feature with the `PILOT_ENABLE_DESTINATION_RULE_INHERITANCE` environment variable.
```
Comment (ericvn) [inline content/en/news/releases/1.9.x/announcing-1.9.1/index.md:52]:
```suggestion
- **Added** support for `DestinationRule` inheritance for mesh/namespace level rules. Enable feature with the `PILOT_ENABLE_DESTINATION_RULE_INHERITANCE` environment variable.
```

File: content/en/news/security/istio-security-2021-001/index.md [hunk 22]
```diff
@@ -0,0 +1,46 @@
...
+---
+
+{{< security_bulletin >}}
+
+This issue only affects Istio 1.9.0; previous versions of Istio are not affected. This issue has been given a CVSS score
+of 8.2 by the Istio product security working group.
+
+Envoy, and subsequently Istio, is vulnerable to a newly discovered vulnerability:
+
+- [Envoy JWT filter bypass when using the allow_missing configuration under requires_any](https://groups.google.com/g/envoy-security-announce/c/aqtBt5VUor0).
```
Comment (ericvn) [inline content/en/news/security/istio-security-2021-001/index.md:21]:
```suggestion
- [Envoy JWT filter bypass when using the `allow_missing` configuration under `requires_any`](https://groups.google.com/g/envoy-security-announce/c/aqtBt5VUor0).
```

File: content/en/news/security/istio-security-2021-001/index.md [hunk 23]
```diff
@@ -0,0 +1,46 @@
...
+{{< warning >}}
+Please note that `RequestAuthentication` is used to define a list of issuers that should be accepted. It does not reject
+a request without JWT token.
+{{< /warning >}}
+
+For Istio, this vulnerability only exists if your service:
+* Accepts JWT tokens (with `RequestAuthentication`)
+* Has some service paths without `AuthorizationPolicy` applied.
+
+If both conditions are met, then an incoming request with a JWT token, and the token issuer is not in
```
Comment (myidpt) [inline content/en/news/security/istio-security-2021-001/index.md:36]:
```suggestion
For the service paths that both conditions are met, an incoming request with a JWT token, and the token issuer is not in
```

**Comment by ericvn** [comment]
.spelling needs v1.29. with the trailing period.

**Pull request merged**